src/components/Dashboard.vue

```diff
@@ -82,49 +97,55 @@ export default {
     },
     stopEvent () {
       event.stopPropagation()
+    },
+    openSendModal (id) {
+      // モーダルウィンドウ表示
+      this.showSendModal = true
+      this.destinationId = id
+    },
+    closeSendModal () {
+      this.showSendModal = false
+    },
+    updateWallet () {
+      // 送信先ユーザの残高取得
+      const createUserArray = this.userList.filter(doc => doc[0] === this.destinationId)
+      const createUser = createUserArray[0]
+      const destinationWallet = createUser[3] + this.sendingMoney
+      const currentWallet = this.wallet - this.sendingMoney
+      async function updateWalletCall () {
+        await store.dispatch('updateWallet', {destinationId: this.destinationId, destinationWallet: destinationWallet, currentId: this.userId, currentWallet: currentWallet})
```

Comment:
async / await を使うならこの箇所は以下のようになります。
関数内で新たに関数を定義する必要はありません。
```suggestion
      await store.dispatch('updateWallet', {destinationId: this.destinationId, destinationWallet: destinationWallet, currentId: this.userId, currentWallet: currentWallet})
```